module/Motion/ArmFalling.h

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file ArmFalling.h
+ * @brief アームを下げるクラス
+ * @author KakinokiKanta
+ */
+
+#ifndef ARM_FALLING_H
+#define ARM_FALLING_H
+
+#include "Motion.h"
+
+class ArmFalling : public Motion {
+ public:
+  /**
+   * コンストラクタ
+   */
```

Comment:
引数についても書いてほしい
```suggestion
  /**
   * コンストラクタ
   * @param _angle アームを下げる角度 0~
   * @param _pwm PWM値 0~100
   */
```
値の範囲はあってるかわかんないから，正しいのにして